application/src/test/resources/persistence.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this is a copy of the non-test file it really shouldn't be needed. The reason it didn't work without it was perhaps that the non-test one is located in META-INF?

Reply:
You are correct! 👍